complex-types.d.ts

```diff
@@ -1,10 +1,10 @@
 import type {Literal} from 'hast'
 
-export type Raw = {
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
```

Comment:
Could `// eslint-disable-next-line @typescript-eslint/consistent-type-definitions` be used to further scope the disable?

Reply:
Yes, I was just following precedent from https://github.com/syntax-tree/mdast-util-to-hast/commit/d97bcbdb5c8841e2a6083100a62c5e1bf8d55d6d. I'll disable it for each interface individually.

Reply:
Done.